spec/controllers/comments_controller_spec.rb

```diff
@@ -2608,7 +2608,17 @@
             end
           end
 
-          %w[superadmin board policy_and_abuse communications support].each do |admin_role|
+          context "with no role" do
+            it_behaves_like "doesn't destroy comment and redirects with error"
+          end
+
+          %w[communications elections].each do |admin_role|
+            context "with role #{admin_role}" do
+              it_behaves_like "doesn't destroy comment and redirects with error"
```

Comment:
The communications and elections roles aren't actually being tested here because the shared example assigns no roles to the admin: `admin.update(roles: [])`. This is also true with the other shared example.

Also, I think there are some roles that aren't being tested, like translation, docs, and open_doors, who also shouldn't be able to delete tag or work comments.